Document credentials update for Enphase Envoy (#31410)
home-assistant/home-assistant.github.io: source/_integrations/enphase_envoy.markdown

### Battery level

When the relay mode is set to battery level, the relays will turn on and off based on the remaining battery level of your Encharge batteries. Two number entities are available to control the cutoff and restore levels for the relays. When the battery level drops below the cutoff level, the relays will turn off. When the battery level rises above the restore level, the relays will turn back on.

## Polling interval

The default polling interval is 60 seconds. To customize the polling interval, refer to [defining a custom polling interval](/common-tasks/general/#defining-a-custom-polling-interval). Specify one single entity from the envoy device as target of the service using the `+ choose entity` button. Updating one entity will update all entities of the Envoy and the related devices like the inverters; there is no need to specify multiple or all entities or add (all) inverter entities. When using multiple Envoys, add one entity for each envoy as targets or create separate custom polling intervals with a single entity as needed.

## Credentials update

> This integration supports changing authentication credentials by adding another envoy integration specifying the same or new IP address, username and password. Use this if you need to update your Enlighten credentials in Home Assistant. The existing entry is detected using the Envoy serial-number obtained from the specified IP address.

Reviewer: ```suggestion
This integration supports updating configuration by re-adding the integration and specifying the same or new IP address, username, and password. Use this method if your Enlighten credentials or the device's IP address has changed and needs to be updated.
```

I tried to leave out the implementation details as the user doesn't need to know how the internal matching works as it might change in the future
